lib/src/pages/home.dart

```diff
@@ -40,7 +85,9 @@ class HomePageState extends State<HomePage>{
 	@override
 	Widget build(BuildContext context) => Scaffold(
 		appBar: AppBar(
+      automaticallyImplyLeading: false,
 			title: Text("Dashboard v${models.home.version ?? ''}"),
+      flexibleSpace: Center(child: Timer()),
```

Comment:
Really good call on the `flexibleSpace`. I don't think I've ever gotten that right before!